sdk/core/core-tracing/src/createSpan.ts

```diff
@@ -33,6 +34,21 @@ export interface CreateSpanFunctionArgs {
   namespace: string;
 }
 
+/**
+ * @internal
+ * A set of known span attributes that will exist on a context
+ */
+export const knownSpanAttributes = {
```

Comment:
Wasn't sure if I should export this for core-tracing. It's something that tracingPolicies will need to know about (for now it's duplicated) and in the future as we add other semantic attributes we will likely need some source of truth for what's available. But..... we'll likely know more then than we do now so I figured let's hold off on exposing this publicly until we know what all the uses are